plugins/ceph/check-ceph.rb

```diff
@@ -92,7 +95,7 @@ def run_cmd(cmd)
     pipe, status = nil
     begin
       cmd += config[:cluster] if config[:cluster]
-      cmd += config[:keyring] if config[:keyring]
+      cmd += config[:user] if config[:user]
```

Comment:
keyring still appears in [documentation](http://docs.ceph.com/docs/jewel/rados/operations/monitoring/) (outdated?)